tpm2/tpm2.go

```diff
@@ -1031,7 +1031,7 @@ func encodeDefineSpace(owner, handle tpmutil.Handle, ownerAuth, authVal string,
 }
 
 // NVDefineSpace creates an index in TPM's NV storage.
-func NVDefineSpace(rw io.ReadWriter, owner, handle tpmutil.Handle, ownerAuth, authString string, policy []byte, attributes uint32, dataSize uint16) error {
+func NVDefineSpace(rw io.ReadWriter, owner, handle tpmutil.Handle, ownerAuth, authString string, policy []byte, attributes NVAttr, dataSize uint16) error {
```

Comment:
This is a (subtly) breaking change for dependents who rely on func signature being stable.

If this PR is blocking some other work, revert just this one line (caller can still use `NVAttr` and convert to `uint32`).
If not, let me try to get https://github.com/google/go-tpm/issues/58 done this or next week and we can submit the PR as is.

Reply:
I'm ok with waiting for this to become a Go module. Not a problem.